### PR TITLE
add doc on how to integrate with COS

### PR DESCRIPTION
**Pull request by kian99**
# Description

This PR introduces a how-to doc on integration JAAS with the Canonical Observability Stack.

The structure, style and wording was heavily lifted from the [Identity platform's how-to](https://charmhub.io/topics/canonical-identity-platform/how-to/integrate-cos) on integrating with COS.

Fixes: [CSS-8612](https://warthogs.atlassian.net/browse/CSS-8612)

[CSS-8612]: https://warthogs.atlassian.net/browse/CSS-8612?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: how-to/integrate_with_cos.rst [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+Prerequisites
+-------------
+
+- A running `COS-Lite`_ bundle. 
+  You can follow the `Getting started on MicroK8s <https://charmhub.io/topics/canonical-observability-stack/tutorials/install-microk8s>`__
+  tutorial to get you started. Make sure to follow the section **Deploy the COS Lite** bundle with overlays sections.
+- A running JAAS. Please refer to the deployment :doc:`tutorial<../tutorial/deploy_jaas_microk8s>`.
+
+It is generally recommended to keep the observability stack separate from any observed applications to separate failure domains. 
+This document assumes that JAAS and the COS bundles are deployed to different models.
```
Comment (alesstimec) [inline how-to/integrate_with_cos.rst]: models, or even k8s clusters

File: how-to/integrate_with_cos.rst [hunk 2]
```diff
@@ -0,0 +1,142 @@
...
+
+This how-to assumes that Vault and PostgreSQL are deployed alongside JIMM and OpenFGA. Depending on your approach, this may not be true.
+Additionally this how-to assumes the names of the deployed applications, these might differ in your environment.
```
Comment (alesstimec) [inline how-to/integrate_with_cos.rst]: nitpick: s/these might/which might/

File: how-to/integrate_with_cos.rst [hunk 3]
```diff
@@ -0,0 +1,142 @@
...
+
+Grafana integration 
+^^^^^^^^^^^^^^^^^^^
+
+Assuming you deployed the COS-Lite bundle in model ``cos-model`` with user admin, use the following 
+commands to integrate the JAAS applications by means of an application offer.
+
+.. code:: bash
+
+    juju integrate jimm admin/cos-model.grafana-dashboards
```
Comment (alesstimec) [inline how-to/integrate_with_cos.rst:51]: this also assumes that application offers were created, right? is that step described anywhere?

Comment (alesstimec) [reply how-to/integrate_with_cos.rst:51]: ah, yes, i see above.. maybe we should emphasize that the offers overlay is required?

**Review by alesstimec — APPROVED** [review]
LGTM with a few nitpicks

File: how-to/integrate_with_cos.rst (hunk 1 above)
Comment (kian99) [reply how-to/integrate_with_cos.rst]: I'll just leave it as models to keep it simple.

File: how-to/integrate_with_cos.rst (hunk 3 above)
Comment (kian99) [reply how-to/integrate_with_cos.rst:51]: Added a "hint" section on this above.

**Pull request merged**